lib/bap_image/objectFile.mli

```diff
@@ -0,0 +1,50 @@
+(** LLVM object file *)
+
+open Core_kernel.Std
+
+type t
+
+external create : Bigstring.t -> t Or_error.t =
+  "object_file_create_stub"
+
+type archType =
```

Comment:
no camelCase, please. And what is the reason to implement module `Arch`? Prune this type.